etc/pylintrc

```diff
@@ -321,4 +321,4 @@ exclude-protected=_asdict,_fields,_replace,_source,_make
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "Exception"
-overgeneral-exceptions=Exception
+overgeneral-exceptions=builtins.Exception
```

Comment:
i'd favor disabling this - when I throw a general exception it's to crash the program.

Reply:
I think it has some merit. The reasoning behind the pylint complaint is that it can result in programmers being incentivised to catch the general `Exception` instead of letting the program crash. Using one of the more specific errors narrows that down and prevents accidental failures in that case. There's also the built-in `RuntimeError` that gets the closest to "please crash".

In the long run, we should prefer implementing our own errors anyway (see https://github.com/SFTtech/openage/issues/1344).